chapters/functions.tex

```diff
@@ -1333,7 +1333,7 @@ \subsection{Using the Derivative Annotation}\label{using-the-derivative-annotati
 \end{lstlisting}
 \end{example}
 
-The inputs to the derivative function of {\lstinline!order!} 1 are constructed as
+The inputs and outputs to the derivative function of {\lstinline!order!} 1 are constructed as
```

Comment:
```suggestion
The inputs and outputs of the derivative function of {\lstinline!order!} 1 are constructed as
```
To me 'to' only works for inputs, but not for outputs, and 'of' seems to work for both.